<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarForm</class>
 <widget class="QWidget" name="CarForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Ð¡haracteristics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Color"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Country"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>EngineName</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="Price"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="EngineName"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ModelName</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>EngineVolume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="ModelName"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="EngineVolume"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mass</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>WheelDrive</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="Mass"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="WheelDrive"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lenght</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>TransmissionType</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="Lenght"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="TransmissionType"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>HorsePower</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="Width"/>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="HorsePower"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>MaxSpeed</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="Height"/>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="MaxSpeed"/>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Clearance</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>BodyType</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLineEdit" name="Clearance"/>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="BodyType"/>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>ManufacturerName</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLineEdit" name="ManufacturerName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ApplyEdit">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Ok</sender>
   <signal>clicked()</signal>
   <receiver>CarForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
